CONTRIBUTING.md

```diff
@@ -27,7 +27,26 @@ Encountering challenges is a normal part of the process. If you need help:
 ## Contributing Guidelines
 
 - **Commit Messages:** Use concise and descriptive commit messages. This helps with the project's history and review process.
-- **Pull Requests:** Before submitting a pull request, make sure your code complies with the project's standards and passes all tests by running `dune fmt` and `dune runtest`. Note you will need `ocamlformat=0.26.1`.
+- **Pull Requests:** Before submitting a pull request, make sure your code complies with the project's standards and passes all tests.. This process involves two key steps:
+  1. **Code Formatting:** Use `dune fmt` to automatically format your code according to the project's style guidelines. This command will apply the formatting rules defined in the `.ocamlformat` configuration file, ensuring consistency across the project.
```

Comment:
The text a bit too long and repeats `dune fmt` twice. Is it necessary to talk about `.ocamlformat` files ?

Reply:
Truly, there is need to talk about `.ocamlformat` here.
I will remove unnecessary information here to aid clarity.

Thank you.